frontend/preview/development/releasing.html

```diff
@@ -3,7 +3,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>evy docs · Releasing</title>
```

Comment:
I think `extractHeading()` is now broken. It type asserts blocks to `(*markdown.Heading)` and I guess they are not that any more, even though your new heading embeds this.

Reply:
yeah, need to fix this, 💰